src/main/java/server/FileResourceHandler.java

```diff
@@ -43,4 +50,35 @@ private void writeContentToFile(String resourceIdentifier, byte[] content) throw
         fileOutputStream.flush();
         fileOutputStream.close();
     }
+
+    public String getDelimitedContentsOfDirectory() {
```

Comment:
Is this used by anything outside of this class? could it be private?

Reply:
Good catch, absolutely can be private.